reference/library/src/webapp/js/spinner.js

```diff
@@ -53,7 +53,7 @@ SPNR.insertSpinnerAfter = function( clickedElement, escapeList, overrideSpinnerL
     spinner.className = "spinPlaceholder";
 
     // If an override location for the spinner is desired, append it as a child...
-    if( overrideSpinnerLocation !== null && overrideSpinnerLocation !== "" )
+    if( overrideSpinnerLocation !== null && overrideSpinnerLocation !== "" && document.getElementById( overrideSpinnerLocation ) )
```

Comment:
Should it be sufficient to say:
if( overrideSpinnerLocation && document.getElementById( overrideSpinnerLocation )